draft-ietf-quic-http.md

```diff
@@ -241,6 +241,9 @@ frame:
   frames."  References without this preface refer to frames defined in
   {{frames}}.
 
+HTTP/3 connection:
+: A QUIC connection where the negotiated application protocol is HTTP/3.
```

Comment:
```suggestion
: A QUIC connection where the selected application protocol is HTTP/3.
```

make it more consistent with the notion of H3 selection in #4211

Reply:
Huh, I always thought negotiated was preferred since we're using ALPN?

Reply:
The spec also says that other methods of selecting H3 are possible. And if we're speaking generically, then that might not be a negotiation. But I'll let Mike make the editorial decision here.

Reply:
Either works; but as ALPN involves a server selection and the N stands for negotiation, I don't think it matters.

Reply:
I'm inclined toward keeping negotiation here.